terraform/modules/hub/hub_policy.tf

```
@@ -172,3 +259,53 @@ module "policy_can_connect_to_ingress_for_metadata" {
   source_sg_id      = module.policy_ecs_asg.instance_sg_id
   destination_sg_id = aws_security_group.ingress.id
 }
+
+module "policy_fargate_can_connect_to_config_fargate_v2" {
+  source = "./modules/microservice_connection"
+
+  source_sg_id      = module.policy_fargate.task_sg_id
+  destination_sg_id = module.config_fargate_v2.lb_sg_id
+}
+
+module "policy_fargate_can_connect_to_saml_engine_fargate" {
+  source = "./modules/microservice_connection"
+
+  source_sg_id      = module.policy_fargate.task_sg_id
+  destination_sg_id = module.saml_engine_fargate.lb_sg_id
+}
+
+module "policy_fargate_can_connect_to_saml_proxy_fargate" {
+  source = "./modules/microservice_connection"
+
+  source_sg_id      = module.policy_fargate.task_sg_id
+  destination_sg_id = module.saml_proxy_fargate.lb_sg_id
+}
+
+module "policy_fargate_can_connect_to_saml_soap_proxy" {
+  source = "./modules/microservice_connection"
+
+  source_sg_id      = module.policy_fargate.task_sg_id
+  destination_sg_id = module.saml_soap_proxy.lb_sg_id
+}
+
+module "policy_can_connect_to_saml_soap_proxy_fargate" {
```

```suggestion
module "policy_fargate_can_connect_to_saml_soap_proxy_fargate" {
```

Terraform may not like having duplicate modules (policy_can_connect_to_saml_soap_proxy_fargate).